asv_bench/benchmarks/rolling.py

```diff
@@ -37,10 +37,14 @@ def setup(self, *args, **kwargs):
     def time_rolling(self, func, center):
         getattr(self.ds.rolling(x=window, center=center), func)()
 
-    @parameterized(['func', 'center'],
+    @parameterized(['func', 'pandas'],
                    (['mean', 'count'], [True, False]))
-    def time_rolling_long(self, func, center):
-        getattr(self.da_long.rolling(x=window, center=center), func)()
+    def time_rolling_long(self, func, pandas):
+        if pandas:
+            se = self.da_long.to_series()
+            getattr(se.rolling(window=window), func)()
+        else:
+            getattr(self.da_long.rolling(x=window), func)()
```

Comment:
Also added a benchmark using pandas for long 1d-array, in order to make it easier to find the cause of the regression.
But is it too verbose as pandas might make the similar benchmark tests.

Reply:
I'm OK with this.